WordPress/WordPressStatsWidgets/LockScreenWidgets/Models/LockScreenWidgetViewModelMapper.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+struct LockScreenWidgetViewModelMapper {
+    let data: HomeWidgetData
+
+    func getLockScreenSingleStatViewModel(title: String, dateRange: String) -> LockScreenSingleStatViewModel {
+        LockScreenSingleStatViewModel(
+            siteName: getSiteName(),
+            title: title,
+            value: getViews(),
+            dateRange: dateRange,
+            updatedTime: data.date
+        )
+    }
+
+    func getStatesURL() -> URL? {
+        if let todayData = data as? HomeWidgetTodayData {
```

Comment:
Okay, since we're building the feature step by step it's good to define the scope as clearly as possible in the description. Since getStatsURL and getViews differed I didn't know what the intention was.